.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint
```

Comment:
Once you have this working, we should figure out how to make this a generic "github action" that anyone can use?

Reply:
It does work right now - it's just a lot of pre-existing linting issues :-).
A centralized github action sounds sweet!